src/compiler/diagnosticMessages.json

```diff
@@ -1364,6 +1364,26 @@
         "category": "Error",
         "code": 1433
     },
+    "'{0}' is a type and must be imported with a type-only import when 'importsNotUsedAsValues' is set to 'preserve-exact'.": {

```

Comment:
The predecessors to these messages try to say `'import type'` and `'export type'` instead of `type-only import` and `type-only export`, but these were worded with an eye toward allowing type-only import and export _specifiers_, as mentioned in the PR description. If we decide not to do that, I would reword these to say `'import type'` and `'export type'` since there would only be one syntax that can fix the error.

Reply:
Also, the imported thing with this error could technically be an uninstantiated namespace, so “is a type” might be technically incorrect, but I think I prefer calling an uninstantiated namespaces a “type” to calling it an “uninstantiated namespace” in user-facing messages.

Reply:
- is a type
- is a type-only entity
- is only used for type constructs
- is only used for type-checking
- is a typey boi

Reply:
"has no runtime representation"

Reply:
Or, more specifically:

`"'{0}' is a type or namespace that does not have an associated runtime value. It must be declared as a type-only import when 'importsNotUsedAsValues' is set to 'preserve-exact'."`